non-free.md

```diff
@@ -87,6 +87,7 @@
 ### Photo and Video Galleries
 
 - [Chevereto](https://chevereto.com/) - Powerful and fast image hosting script that allows you to create your very own full featured image hosting website in just minutes. ([Demo](https://demo.chevereto.com/)) `⊘ Proprietary` `PHP`
+- [gScreen](https://photoslideshow.app/) - Shuffled slideshows across your Google Photos libraries, viewable on any device with a web browser. ([Demo](https://demo.photoslideshow.app)) `⊘ Proprietary` `Go`
```

Comment:
> Google Photos libraries,

>  Additions that depend on proprietary services outside the user's control must be marked ⚠.

- Could you add `⚠`  just after the main URL

- [gScreen](https://photoslideshow.app/) `⚠` -  .....